src/components/Docs/Button.js

```diff
@@ -0,0 +1,91 @@
+export const Button = () => {
+  return (
+    <div className="component">
+      <h1>Button</h1>
+      <p className="component-text">
+        The Button component is used to trigger an action or event, such as
+        submitting a form, opening a dialog, canceling an action, or performing
+        a delete operation.
+      </p>
+
+      <div className="component-sub">
+        <h3>Primary Buttons</h3>
+        <div class="flex-row justify-space-evenly component-box">
+          <button class="button button-primary shadow"> Home </button>
+          <button class="button button-primary shadow"> About </button>
+          <button class="button button-primary shadow"> Contact </button>
+        </div>
+
+        <iframe
+          className="component-iframe"
+          src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=one-light&wt=bw&l=auto&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=143%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522button%2520button-primary%2520shadow%2522%253E%2520Home%2520%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522button%2520button-primary%2520shadow%2522%253E%2520About%2520%253C%252Fbutton%253E"
+          sandbox="allow-scripts allow-same-origin"
+        ></iframe>
+      </div>
+
+      <div className="component-sub">
+        <h3>Secondary Buttons</h3>
+        <div class="flex-row justify-space-evenly  component-box">
+          <button class="button button-secondary shadow"> Home </button>
```

Comment:
I must admire, you have taken good care of every small details in your code. one of them is spacing around button names. as it's been smooth experience to going through your code.